src/main/java/org/jnode/fs/xfs/XfsEntry.java

```diff
@@ -30,7 +30,10 @@
      */
     private static final Logger log = LoggerFactory.getLogger(BlockDirectoryEntry.class);
 
-    long NANO_SECOND = 1000000000;
+    /**
+     * Constant used to convert nano seconds to milliseconds.
+     */
+    private static final long ONE_MILLISECOND = 1_000_000;
```

Comment:
The constant name isn't quite right. It's strange to say that `ONE_MILLISECOND` is 1,000,000. Suggestion below.

Reply:
HI , Erwin your suggestion was applied . Thanks